tests/unit/anchore_engine/test_utils.py

```diff
@@ -92,6 +99,58 @@ def test_parse_dockerimage_string(image, expected):
     assert result == expected
 
 
+def test_run_sanitize_good_input():
+    # Setup input
+    input_cmd_list = ["wc", "-l"]
+
+    # Function under test
+    output_cmd_list = run_sanitize(input_cmd_list)
+
+    # Validate input
+    output_cmd_list == input_cmd_list
+
+
+@pytest.mark.parametrize(
+    "input",
+    [";&<>", ";", "&", "<", ">"],
+)
+def test_run_sanitize_bad_input(input):
+    with pytest.raises(Exception):
+        run_sanitize(input)
+
+
+@pytest.mark.parametrize(
+    "cmds_list, expected_return_code, expected_stdout, expected_stderr",
+    [([], None, None, None), ([["pwd"], ["wc", "-l"]], 0, "1", b"")],
+)
+def test_run_piped_command(
+    cmds_list, expected_return_code, expected_stdout, expected_stderr
+):
+    # Function under test
+    return_code, stdout, stderr = run_piped_command_list(cmds_list)
+
+    # Binary string returned in different environments can be padded with different amounts of whitespace
+    # So where expected_output != None, convert it to utf-8 and trim it so we get a clean, reliable comparison
+    if stdout is not None:
```

Comment:
I think it is very odd to have the function return `None, None, None` - this type of interaction with that function complicates how to deal with the output. I think it would be best to always return the expected values or raise an exception